en/ch02.md

```diff
@@ -66,29 +88,11 @@ Restoring from keys is quite easy and can be extremely useful, especially if you
 (2) **Account address:** The address of the wallet you are recovering.  
 (3) **View key:** Your private view key (needed to be able to check your funds).  
 (4) **Spend key:** Your private spend key (needed to spend your funds).  
-(5) **Restore height (optional):** If you know the block height that your wallet was created at, you can specify it here so the wallet doesn't have to scan the entire blockchain looking for your funds. For example, if your first transaction was included in block `1350000`, you should put a slightly lower height (e.g. `1330000`) so the wallet will start scanning from there, saving you some time.  
+(5) **Creation Date or Restore height (optional):** If you know the date or block height that your wallet was created at, you can specify it here so the wallet doesn't have to scan the entire blockchain looking for your funds. For example, if your first transaction was included on `2017-07-08` in block `1350000`, you should put the previous day date (e.g. `2017-07-06`) or a slightly lower height (e.g. `1330000`) so the wallet will start scanning from there, saving you some time.  
```

Comment:
`Creation date` will be available for hardware Monero wallets as well. Perhaps make sure to account for that.

Reply:
ok